src/ol/render/canvas/CurveBuilder.js

```diff
@@ -270,7 +271,6 @@ class CanvasCurveBuilder extends CanvasBuilder {
     }
     state.lastStroke = 0;
     super.applyStroke(state);
-    this.instructions.push(beginPathInstruction);
```

Comment:
Ik begrijp deze functie nog niet helemaal. Zou, de base functie volstaan? Dan zou deze functie in zijn geheel mogelijk kunnen verdwijnen. Dat `lastStroke` gedoe heeft vast iets met optimalisatie van het één of ander te doen, maar misschien nog niet heel relevant. Zeker als we niet weten hoe het precies zou moeten werken.

Reply:
Als ik de functie weg haal en alleen de base functie gebruik wordt de curve polygon met bijbehorende strokes nog gewoon correct getekend, maar de circular string en compound curve die worden dan niet meer getekend.
Bij het tekenen van de curve polygon komt 'ie wel in de `if` statement en bij de circular string en compound curve niet.
Aangezien de curve polygon wel getekend wordt als die hele functie er niet is, zou de `if` statement dus uit de functie kunnen. Maar goed, dan zijn we een beetje half bezig en aangezien de curve polygon wél weer in de if komt (geen idee wat dat dan betekend) kunnen we het dan denk ik maar beter zo laten?

Reply:
Ja, laat maar zo in dat geval. Ooit misschien nog eens in verdiepen ...